browser/IgBlazorSamples.Gulp/tasks/gulp-samples.js

```diff
@@ -492,7 +491,13 @@ function copySampleScripts(cb, outputPath, indexName) {
 // '../../browser/IgBlazorSamples.Server/wwwroot'
 function copySamplesToServer(cb) {
     cleanupSampleBrowser( "../../browser/IgBlazorSamples.Server");
-    copySampleScripts(cb, "../../browser/IgBlazorSamples.Server", "/Pages/_Host.cshtml");
+    copySampleScripts(cb, "../../browser/IgBlazorSamples.Server", "/Pages/_Host.cshtml", true);
+    copySamplePages(cb,   "../../browser/IgBlazorSamples.Server");
+} exports.copySamplesToServer = copySamplesToServer;
+
+function copySamplesToServerHost(cb) {
```

Comment:
Okay, this is better than the previous hack, though still would like to not perpetuate the practice of describing nothing at all :D Whatever "to server host" means I doubt it'll make sense to the next person looking at this. At the very least `copySamplesToServerCI` or something?
If we're doing this, we're also adjusting the pipelines that build & deploy this internally, no? If so, this could be a single command with a param for the base path (default `/`) since that's supposed to come from the external CI anyway...
(PS: gulp doesn't provide args which is a bit silly, but throwing in `yargs` is easy enough)